README.md

```diff
@@ -2,92 +2,133 @@
 
 ## Introduction
 
-The "Portal Network" is an in progess effort to enable lightweight protocol access to resource constrained devices.  The term *"portal"* is used to indicate that these networks provide a *view* into the protocol but are not critical to the operation of the core Ethereum protocol.
+The Portal Network is an in progess effort to enable lightweight protocol access to resource constrained devices.  The term *"portal"* is used to indicate that these networks provide a *view* into the protocol but are not critical to the operation of the core Ethereum protocol.
 
+The Portal Network is comprised of multiple peer-to-peer networks which together provide the data and functionality necessary to expose the standard JSON-RPC API.  These networks are specially designed to ensure that clients participating in these networks can do so with minimal expendature of networking bandwidth, CPU, RAM, and HDD resources.
```

Comment:
I'm not sure what the intended audience for this document is but a link to the api might be useful to them: https://eth.wiki/json-rpc/API

Reply:
s/expendature/expenditure/